cmd/bootstrap/run/qc.go

```diff
@@ -38,24 +38,18 @@ func (pd *ParticipantData) Identities() flow.IdentityList {
 	return bootstrap.ToIdentityList(nodes)
 }
 
-func GenerateRootQC(block *flow.Block, participantData *ParticipantData) (*flow.QuorumCertificate, error) {
-
+// GenerateRootQC generates QC for root block, caller needs to provide votes for root QC and
+// participantData to build the QC.
+// NOTE: only one DKG private key is required to sign the QC, for all others entries we can
+// provide only identity info without private keys.
```

Comment:
A QC is not signed. Lets update this:

```suggestion
// NOTE: at the moment, we require private keys for one node because we we re-using the full business logic, which assumes that only consensus participants construct QCs, which also have produce votes. 
// TODO: modularize QC construction code (and code to verify QC) to be instantiated without needing private keys. 
```